libs/stdlib/iec61131-st/to_num.py

```diff
@@ -0,0 +1,65 @@
+f = open("to_x.txt", "w")
+
+types = [
+    "SINT",
+    "USINT",
+    "INT",
+    "UINT",
+    "DINT",
+    "UDINT",
+    "LINT",
+    "ULINT",
+    "LREAL",
+    "REAL",
+]
```

Comment:
It seems like we are missing quite a few types here, i.e. `TIME`, `LTIME`, `LWORD`, (`STRING`?), etc.  I've tried converting a `TIME` variable to `USINT`, which will just end up calling `TO_USINT__USINT`, silently truncating the input down to 8 bits, losing most of the information and returning wrong values.
Are these planned for a different commit?

Reply:
I'll have to check what CodeSys supports here, but `BOOL`(?), `BYTE`, `WORD`, `DWORD` and `LWORD` are (probably) missing here. I forgot about them somehow